packages/components/src/components/video/video.stories.ts

```diff
@@ -47,31 +150,45 @@ export const VideoElement = {
   },
   render: (args: any) => {
     return html`
-      ${generateTemplate({
-        args,
-        constants: {
-          type: 'slot',
-          name: 'default',
-          value:
-            '<video controls poster="https://www.blender.org/wp-content/uploads/2020/10/robin-tran-redautumnforest_pr1.jpg" style="max-width: 854px; may-height: 480px;"><source src="http://media.w3.org/2010/05/sintel/trailer.mp4" type="video/mp4" />Your browser does not support the video tag.</video>'
-        }
-      })}
-      <script>
-        const videoEl = document.querySelector('video');
-        const sdVideo = document.querySelector('sd-video');
-
-        sdVideo.addEventListener('sd-play', () => {
-          videoEl.play();
-        });
-
-        videoEl.addEventListener('play', () => {
-          sdVideo.play();
-        });
-
-        videoEl.addEventListener('pause', () => {
-          sdVideo.playing = false;
-        });
-      </script>
+      <div id="sdVideoExample">
+        ${generateTemplate({
+          args,
+          constants: [videoConstant, posterConstant]
+        })}
+        ${videoElementScript}
+      </div>
     `;
   }
 };
+
+/**
+ * Duplicates the previous story with default focus on the play button.
```

Comment:
we need to deactivate the mouse in this story, when hovering over the component. Have a look into the sd-tag mouseless story to see how its implemented.